src/Pages/Main/Components/Submenu/SubCardA.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import styled from "styled-components";
+import { font, theme } from "../../../../Styles/GlobalStyles";
+
+function SubCardA({ product }) {
+  if (product) {
+    return (
+      <SubCardStyle>
+        <SubCardImg src={product.image} />
+        <SubCardTitle>{product.name}</SubCardTitle>
+        <SubCardBox>
+          <SubCardIcon />
+          <CardText>
+            <p>{product.sub_name}</p>
+            For <span>{product.editor}</span> | <span>From </span>
+            <span>{`$${product.price[0]}`}</span>
+            <span>{product.price[1] ? `$${product.price[1]}` : null}</span>
```

Comment:
```suggestion
            // const [fooStr, barStr] = product.price
            
            <span>{"$" + fooStr}</span>
            <span>{barStr && "$" + barStr}</span>
```
- 배열도 구조분해할당 해줄 수 있습니다. 이 경우 새로운 변수명을 지어줄 수 있기 때문에 가독성이 개선됩니다
- $ 기호를 써주시고 싶으셨던 것 같은데 이 경우 $$가 연속으로 붙어 있는 것이 어색해보입니다 그냥 + 기호를 써서 붙여주는게 좋아보입니다
- true 값만 필요할 경우 &&로 써주세요

Reply:
수정했습니다!